src/client/component.js

```diff
@@ -3,8 +3,7 @@ const walkVars = require('./visitors/vars');
 const walkNode = require('./visitors/node');
 const utils = require('./utils');
 
-require('__IDYLL_SYNTAX_HIGHLIGHT__');
-let results = require('__IDYLL_AST__');
+// require('__IDYLL_SYNTAX_HIGHLIGHT__');
```

Comment:
Yeah, that would work, I think. *Possible* via a conditional require and then `envify` to strip it out so that it's removed for the client bundle. Otherwise might have to do the same trick and pass it down from the top.